overrides/scripts/ThermalExpansion.zs

```diff
@@ -484,4 +484,17 @@ furnace.remove(<thermalfoundation:rockwool:7>, <thermalfoundation:material:864>)
 
 //Satchel Removal
 recipes.removeByRecipeName("thermalexpansion:satchel_1");
-recipes.removeByRecipeName("thermalexpansion:satchel_7");
\ No newline at end of file
+recipes.removeByRecipeName("thermalexpansion:satchel_7");
+
+//Pure Certus
+mods.thermalexpansion.Insolator.addRecipe(<appliedenergistics2:material:10>, <appliedenergistics2:crystal_seed>.withTag({progress: 0}), <minecraft:glowstone_dust>, 40000);
+
+//Pure Nether Quartz
+mods.thermalexpansion.Insolator.addRecipe(<appliedenergistics2:material:11>, <appliedenergistics2:crystal_seed:600>.withTag({progress: 600}), <minecraft:glowstone_dust>, 40000);
+
+//Pure Fluix
+mods.thermalexpansion.Insolator.addRecipe(<appliedenergistics2:material:12>, <appliedenergistics2:crystal_seed:1200>.withTag({progress: 1200}), <minecraft:glowstone_dust>, 40000);
+
+mods.jei.JEI.addDescription(<appliedenergistics2:material:10>, "Made in the Crystal Growth Chamber or in a Phytogenic Insolator. If made in the Phytogenic Insolator, make sure to unlock the Fertilizer slot. Augments do not work for this craft.");
+mods.jei.JEI.addDescription(<appliedenergistics2:material:11>, "Made in the Crystal Growth Chamber or in a Phytogenic Insolator. If made in the Phytogenic Insolator, make sure to unlock the Fertilizer slot. Augments do not work for this craft.");
+mods.jei.JEI.addDescription(<appliedenergistics2:material:12>, "Made in the Crystal Growth Chamber or in a Phytogenic Insolator. If made in the Phytogenic Insolator, make sure to unlock the Fertilizer slot. Augments do not work for this craft.");
```

Comment:
These create info tabs for the items in JEI. Do they override an existing one from AE2? I know fluix crystals get one but I forget if other crystal types do as well.

Also worth noting that we assign a custom tooltip on these items in `AE2.zs`. I wonder if that has anything to do with the unusual Phytogenic Insolator behavior?

Reply:
I tested without the custom tooltip, and the original recipes, and it still did not work. The recipes added by this PR work with the custom tooltip, so I believe we can rule out the tooltip as a source of error. As for the info tab, the existing one from AE2 is not overwritten, but instead a new one is created.
![crystal](https://user-images.githubusercontent.com/31759736/87865013-5e84f980-c924-11ea-82a6-acb6a492e14c.PNG)

Reply:
Thank you for clarifying, and I trust that you've investigated this quite thoroughly. I want to make sure that the PR documents clearly why we took the approach we did so we can refresh our memory in the future (and also so we don't need to rely on links to discord conversations for context).